README.md

```diff
@@ -1,283 +1,487 @@
+
+### Module Formats
+
+webpack supports ES2015+, CommonJS and AMD modules **out of the box**. It performs clever static
 analysis on the AST of your code. It even has an evaluation engine to evaluate
 simple expressions. This allows you to **support most existing libraries** out of the box.
 
-## Code Splitting
+### [Code Splitting](https://webpack.github.io/docs/code-splitting.html)
 
 webpack allows you to split your codebase into multiple chunks. Chunks are
 loaded asynchronously at runtime. This reduces the initial loading time.
 
-[Code Splitting documentation](https://webpack.github.io/docs/code-splitting.html)
-
-## Optimizations
+### [Optimizations](https://webpack.github.io/docs/optimization.html)
 
 webpack can do many optimizations to **reduce the output size of your
 JavaScript** by deduplicating frequently used modules, minifying, and giving
 you full control of what is loaded initially and what is loaded at runtime
 through code splitting. It can also make your code chunks **cache
 friendly** by using hashes.
 
-[Optimization documentation](https://webpack.github.io/docs/optimization.html)
-
-# A small example of what's possible
-
-``` javascript
-// webpack is a module bundler.
-// This means webpack takes modules with dependencies
-// and emits static assets representing those modules.
-
-// Dependencies can be written in CommonJS
-var commonjs = require("./commonjs");
-// or in AMD
-define(["amd-module", "../file"], function (amdModule, file) {
-	// while previous constructs are sync,
-	// this is async
-	require(["big-module/big/file"], function (big) {
-		 // For async dependencies, webpack splits
-		 // your application into multiple "chunks".
-		 // This part of your application is
-		 // loaded on demand (code-splitting).
-		var stuff = require("../my/stuff");
-		// "../my/stuff" is also loaded on-demand
-		//  because it's in the callback function
-		//  of the AMD require.
-	});
-});
-
-
-require("coffee!./cup.coffee");
-// "Loaders" are used to preprocess files.
-// They can be prefixed in the require call
-// or configured in the configuration.
-require("./cup");
-// This does the same when you add ".coffee" to the extensions
-// and configure the "coffee" loader for /\.coffee$/
-
-function loadTemplate (name) {
-	return require("./templates/" + name + ".jade");
-	// Many expressions are supported in require calls.
-	// A clever parser extracts information and concludes
-	// that everything in "./templates" that matches
-	// /\.jade$/ should be included in the bundle, as it
-	// can be required.
-}
-
-
-// ...and you can combine everything.
-function loadTemplateAsync (name, callback) {
-	require(["bundle?lazy!./templates/" + name + ".jade"],
-	  function (templateBundle) {
-	          templateBundle(callback);
-	});
-}
-```
-
-## Documentation
-
-[documentation](https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=documentation)
+<h2 align="center">Example</h2>
 
+**app.js**
```

Comment:
Same here. Would it be better to defer to the docs to slim this down? Easier to maintain again.